src/components/headroom/headroom.js

```diff
@@ -75,47 +75,25 @@ define(['dom', 'layoutManager', 'browser', 'css!./headroom'], function (dom, lay
         this.pinnedClass = options.pinnedClass;
 
         this.state = 'clear';
-    }
 
-    function onScroll() {
+        this.options = {
+            offset: 0,
+            scroller: window,
+            initialClass: 'headroom',
+            unPinnedClass: 'headroom--unpinned',
+            pinnedClass: 'headroom--pinned'
+        };
 
-        if (this.paused) {
-            return;
-        }
-
-        requestAnimationFrame(this.rafCallback || (this.rafCallback = this.update.bind(this)));
-    }
-
-    Headroom.prototype = {
-        constructor: Headroom,
```

Comment:
What is this file used for? Not a review, just a question.

Reply:
It gives your header dynamic behavior. See for yourself in the plugin official page: http://wicky.nillia.ms/headroom.js/